### PR TITLE
Launch scripts refactor

### PR DESCRIPTION
**Pull request by shamsimam**
Independent changes to declutter diff in cook-executor PR

### CODE REVIEW

**Comment by shamsimam** [comment]
Passing build: https://travis-ci.org/twosigma/Cook/builds/228531626

File: travis/minimesos [hunk 1]
```diff
@@ -48,9 +55,9 @@ fi
 docker run --rm -v "${MINIMESOS_HOST_DIR}":"${MINIMESOS_HOST_DIR}" \
        -v /var/run/docker.sock:/var/run/docker.sock \
        -v /sys/fs/cgroup:/sys/fs/cgroup \
-       -v ${DOCKER_BIN}:/usr/bin/docker \
        -i \
        --env DOCKER_HOST_IP=${DOCKER_HOST_IP} \
+       --env MINIMESOS_OS="${MINIMESOS_OS}" \
```
Comment (dposada) [inline travis/minimesos]: What effect does this have?

**Comment by pschorf** [comment]
LGTM, @dposada can merge after his comment is addressed.

File: travis/minimesos (hunk 1 above)
Comment (shamsimam) [inline travis/minimesos]: I just copy-pasted the latest minimesos file from https://github.com/ContainerSolutions/minimesos/blob/0.13.0/bin/minimesos

Comment (shamsimam) [inline travis/minimesos]: I copy-pasted the content from https://github.com/ContainerSolutions/minimesos/blob/0.13.0/bin/minimesos

**Pull request merged**